drivers/input/Kconfig.cf1133

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2024 Kickmaker
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig INPUT_CF1133
+	bool "CF1133 capacitive touch panel driver"
+	default y
+	depends on DT_HAS_SITRONIX_CF1133_ENABLED
+	select I2C
+	help
+	  Enable driver for Sitronix capacitive touch panel
+	  controller. This driver should support CF1133.
+
+if INPUT_CF1133
+
+config INPUT_CF1133_INTERRUPT
```

Comment:
This Kconfig could be avoided by using the devicetree to check if the int_gpios exists for each instance of the touch screen, and automatically adjust to it existing on a per instance basis, using macros like `DT_ANY_INST_HAS_PROP_STATUS_OKAY`.

In this case, all instances of the touchscreen (if there are more than one, maybe not likely) must have the int_gpios present, or the driver will fail to build if only one screen is connected and `INPUT_CF1133_INTERRUPT` is set.

Please update the help to indicate this :) something like 
```
Enable interrupt support (requires all instances of CF1133 have the INT gpio connected).
```

Reply:
Ok done thank you for your insight !